charts/camunda-platform/charts/identity/Chart.yaml

```diff
@@ -9,3 +9,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     version: 12.3.0
     condition: "keycloak.enabled"
+  - name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: 11.x.x
```

Comment:
Is there any reason to use v11? that's the chart version, which has a bit of old psql.

Reply:
in general I'd say we should use the same as in Keycloak v22 chart so it's easy to disable the one shipped with Keycloak and use this common one.

Reply:
No strong reason, I think it was just copy pasta :) - I can update this though